### DISCUSSION TITLE
#28. VATZ biweekly meeting at 2023-02-15 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#396] [General]
### 1. Overall
<img width="1331" alt="image" src="https://user-images.githubusercontent.com/6308023/216180840-65868e67-5f4b-400a-bb6d-7254d75f66b8.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
26 | 17/34 | 50
27 | 2/19 | 10
---

## Discussion

- https://github.com/dsrvlabs/vatz/issues/405

**Comment by xellos00** [comment]
# 2023-02-09

## Stars
16 -> 18
<img width="1248" alt="image" src="https://user-images.githubusercontent.com/6308023/218856646-6b4ec59b-eb7f-488c-8bba-5fb5718ef0cc.png">

<img width="901" alt="image" src="https://user-images.githubusercontent.com/6308023/217941109-2071a342-31cd-4142-b9b7-145b298846a4.png">

<img width="1370" alt="image" src="https://user-images.githubusercontent.com/6308023/219035629-5fbc1161-bce1-4a83-802d-be084ab023a8.png">

**Comment by xellos00** [comment]
# 동요

- #312
- #323  
- #390 
- #394 
- #398 
- #399 
- #405

**Reply by xellos00** [reply]
@xellos00 
- License 
Put DSRV(`dsrv labs Co., Ltd.`) in Apache 2.0
Company `dsrv labs` in full name.

**Comment by rootwarp** [comment]
# 준교
- https://github.com/dsrvlabs/vatz/issues/403

**Comment by skonhwang** [comment]
# 승곤
- https://github.com/dsrvlabs/injective-project/issues/60#issuecomment-1429247175

**Comment by gnongs** [comment]
## 진홍
- No issue

**Comment by heejin-github** [comment]
## 희진
- No issue

**Reply by xellos00** [reply]
@xellos00 
Leave all the logs with same error. 
- #401

**Comment by kim201212** [comment]
## 형기

- No issue

**Reply by xellos00** [reply]
@kim201212 
record all the history on issue
- https://github.com/dsrvlabs/vatz-proto/issues/14
